test/apm_registry.js

```diff
@@ -62,19 +61,19 @@ contract('APMRegistry', accounts => {
 
     it('fails to create empty repo name', async () => {
         return assertRevert(async () => {
-            await registry.newRepo('', repoDev, { from: apmOwner })
+            return await registry.newRepo('', repoDev, { from: apmOwner })
```

Comment:
Since all of our `assert*` helpers are async themselves, these inner functions can just return the promise, e.g.

```js
return assertRevert(() => {
  return registry.newRepo('', repoDev, { from: apmOwner })
})
```

Reply:
Seems like if we go this route then we will have to add in the logic to handle the 0 status everywhere that we call `assertRevert`. So all test that do not have to deal will odd truffle/geth test differences (`base 16 number`). 

It seems like these tests would then have to use a wrapper to check that the receipt status is 0 and throw the error:

```js
const checkRecieptStatus = code => {
    let result = await code()
    if (result.recipt.status == 0) { throw new Error('mock revert') }
}
```

Which makes the test look something like:
```js
return assertRevert(async () => {
    checkRecieptStatus(() => {
        return await registry.newRepo('', repoDev, { from: apmOwner })
    })
})
```